src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DataFlowAnalysis.cs

```diff
@@ -286,11 +286,9 @@ protected DataFlowAnalysis(AbstractAnalysisDomain<TAnalysisData> analysisDomain,
 
                 // Check if we are starting a try region which has one or more associated catch/filter regions.
                 // If so, we conservatively merge the input data for try region into the input data for the associated catch/filter regions.
-#pragma warning disable CA1508 // Avoid dead conditional code - https://github.com/dotnet/roslyn-analyzers/issues/4408
                 if (block.EnclosingRegion?.Kind == ControlFlowRegionKind.Try &&
                     block.EnclosingRegion.EnclosingRegion?.Kind == ControlFlowRegionKind.TryAndCatch &&
```

Comment:
This one is still a bug. But I cannot reproduce in a unit test.

@Evangelink Would you mind taking a look?